tardis/io/logger/colored_logger.py

```diff
@@ -57,7 +66,7 @@ class ColoredLogger(logging.Logger):
     Custom logger class with multiple destinations
     """
 
-    FORMAT = "[$BOLD%(name)-20s$RESET][%(levelname)-18s]  %(message)s ($BOLD%(filename)s$RESET:%(lineno)d)"
+    FORMAT = "[$BOLD{name:20s}$RESET][{levelname:18s}]  \n\t{message:s} ($BOLD{filename:s}$RESET:{lineno:d})"
```

Comment:
If there is the debug/non-debug variables, why does this still exist?

Reply:
This is just because of the way the `__init__` method inside the `ColoredFormatter` Class works. It takes a message (this is the message) & the `use_color` variable boolean. We would need to refactor the `__init__` function for this to be removed.